test/libyul/YulOptimizerTest.cpp

```diff
@@ -347,6 +350,54 @@ TestCase::TestResult YulOptimizerTest::run(ostream& _stream, string const& _line
 		obj.analysisInfo = m_analysisInfo;
 		OptimiserSuite::run(*m_dialect, &meter, obj, true, solidity::frontend::OptimiserSettings::DefaultYulOptimiserSteps);
 	}
+	else if (m_optimizerStep == "memoryEscalator")
+	{
+		yul::Object obj;
+		obj.code = m_ast;
+		obj.analysisInfo = m_analysisInfo;
+		disambiguate();
+		MemoryEscalator::run(*m_context, obj, true);
+	}
+	else if (m_optimizerStep == "fakeMemoryEscalator")
```

Comment:
Should we keep this like it is? It makes for way easier to read and more stable test cases that way...
But we could think about a better way to choose which variables to move other than having them start with ``$``, but seems as good as any other way of marking them to me...

Reply:
Can you make it a test config option?
```
{
	function z() -> a,b,c { let t := 0 }
	sstore(0, f())
	let x, y := g()
}
// ====
// escalate: t
// ----
// step: fakeMemoryEscalator
// {
//     memoryinit(0xa0)
//     function g() -> a, b
//     {
//         a 
```